### DISCUSSION TITLE
Define board buckets and lock them

### DISCUSSION THREAD

**Discussion by NicolasC67** [marcusolsson/obsidian-projects#828] [Help] [answered]
Hello,
I tried to find if that idea had been logged already here, but I did not find it.

I am using **Projects** to manage my projects of course, but my list of people, ideas and so on through the module. Every type of note that can be DB'ed, I play a lot with file properties. But when it comes to create Kanban boards, this view only display buckets based on the existing values of what I have in my DB folder.

It would be really handy to have the possibility to define a list of buckets in a static manner so they stick around even if there is no Obsidian note have a file property matching with it. Therefore, it would help filling in or assigning the right file property by dragging and dropping on top of an empty bucket.

I included a picture of what I am currently doing:

- its a **Projects** board view of all my **Ideas** in my Obsidian vault
- In green at the top all my buckets that I want to use, and that's the definitive list of values for the **PhaseStatus** file property that I use to organize the kanban view
- In orange, I have artificially created dummy ideas files (which are empty but have the right IdeaPhase) just for the purpose of making the appropriate column displayed.

What do you think? Is it clear? Do you think that would be feasible? :)

Thank you

![2024-03-27_18-03-59](https://github.com/marcusolsson/obsidian-projects/assets/50239396/638013b7-f52a-4ba7-b604-38d825350c28)

**Comment by NicolasC67** [comment]
I would actually manage this through the **Board settings** little portlet, in a similar way than what we see in other plugins, with a + button to add all the different buckets we'd like.
![image](https://github.com/marcusolsson/obsidian-projects/assets/50239396/70a42f88-504f-4835-abca-9401610e1aac)

**Comment by Acylation** [comment, marked as answer]
Yeah this is actually supported, my reply in https://github.com/marcusolsson/obsidian-projects/discussions/661#discussioncomment-6855535 FYI. For short, you can add predefined fields via Table view column settings for your board status field. Then the options would persist even if there's no matching note.

The in-progress PR #739 (corresponding to issue #482) aims to provide a more intuitive way to manage the options/board column. Hope these would help!

**Reply by NicolasC67** [reply]
Thanks a lot, i must have missed it yesterday… but i checked again this morning and found it ! That’s awesome !
